epub33/a11y-tech/index.html

```diff
@@ -133,50 +134,28 @@ <h3>Purpose and Scope</h3>
 			<section id="sec-terminology">
 				<h3>Terminology</h3>
 
-				<p>This document uses the following terminology defined in EPUB 3 [[EPUB-3]]:</p>
+				<p>This document uses terminology defined in EPUB 3 [[EPUB-3]]:</p>
 
-				<ul>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-content-document">EPUB Content Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB Creator</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-navigation-document">EPUB Navigation Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-publication">EPUB Publication</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-media-overlay-document">Media Overlays Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-package-document">Package Document</a>
-					</li>
-					<li>
-						<a href="https://www.w3.org/TR/epub/#dfn-epub-reading-system">Reading System</a>
-					</li>
-				</ul>
+				<p>In addition, it uses the <a href="https://www.w3.org/TR/WCAG2/#dfn-assistive-technologies">WCAG 2
+						definition</a> of <dfn data-lt="assistive technologies">Assistive Technology</dfn>.</p>
```

Comment:
Per [xref](https://respec.org/xref/?term=assistive+technologies) the term assistive technologies can be reached via xref as well, directly from WAI-ARIA. Wouldn't it be better to use that rather than redefining the term here? (Same comment goes for the a11y spec)

Reply:
Ya, but we reference it from WCAG and I'm not sure the definitions are interchangeable. They have similarities, but I suspect the ARIA one has been tailored for the cases it specifically covers.

Reply:
Ok, I leave this to you... but it is looks as if _we_ defined the term when you click to it elsewhere.

Reply:
In a sense we are. We're defining that we use the same meaning as WCAG and also explain that AT may be part of reading systems in the note. That difference makes it unique to our specification.

The way it's phrased right now comes from the rewrite to meet ISO's formatting. It should really be defined as a proper term in a `dl`, and have the note put back closer.

Reply:
Do we want to export that definition (which would mean adding the A11y spec to that list of specs, too)?

Reply:
It may create lots of confusions, though...

Reply:
> It may create lots of confusions, though

Yes, let's not export it. We really only need it in the a11y document and the techniques. I've added a dl for it in the latest commits and the techniques document now references across to both a11y-11 and epub-33 for its terminology, using `data-cite` for AT. That's good enough.